app/views/uploads/_measure_detail.html.erb

```diff
@@ -0,0 +1,53 @@
+<% measures.each do |measure| %>
+  <%patient_cache_subs = @patient_cache_all.where({'value.measure_id'=> measure[:id]})%>
+  <%@patient_cache = patient_cache_subs.select{|k| k.value[:sub_id]==measure[:sub_id]}.first%>
```

Comment:
I think the assignments above would be better suited as helper functions in `app/helpers/uploads_helper.rb`, since they're performing db queries.